smithy-aws-traits/src/main/java/software/amazon/smithy/aws/traits/ServiceTrait.java

```diff
@@ -93,6 +102,15 @@ public static Builder builder() {
         return new Builder();
     }
 
+    /**
+     * Get the ShapeId of the target of this trait.
+     *
+     * @return Returns the ShapeId which this trait targets.
+     */
+    public ShapeId getTarget() {
```

Comment:
```suggestion
    private ShapeId getTarget() {
```

We do not need to and should not expose this. Better to set the default value in the `build` method honestly.

Reply:
That's fair, I added it mostly to consistency, making sure all fields have a public getter, but I don't mind changing it at all as there's no clear need for it.